src/components/stickable/list/StickableCard.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { RouterLink } from 'vue-router';
+import { ref, onMounted, onBeforeUnmount, nextTick } from 'vue';
+
+const props = defineProps<{
+  to: string; // when present, render as RouterLink
+  class: string;
+}>();
+
+const textElement = ref<HTMLElement>();
+const isOverflowing = ref(false);
+
+const checkOverflow = () => {
```

Comment:
Please let me know if this is a very hacky solution.. It was created by my fellow AI-companion, but since I don't have a smarter solution I just used it..